docs/changes.rst

```diff
@@ -27,6 +27,8 @@ Released: not yet
 * Extend use of general options in interactive mode to allow setting the
   connections-file for an interactive command. (see issue #1037)
 
+* Change DOCKER
```

Comment:
This change log entry could be more specific.

Reply:
Not sure what you want for info here.  We are trying to document the build characteristics both in the container and avalable documentation but this is the first release and the only real use for this release is the pywbemtools tests today.  I added mode documentation.